app/controllers/hyrax/my_controller.rb

```diff
@@ -19,6 +19,7 @@ def self.configure_facets
         config.add_facet_field IndexesWorkflow.suppressed_field, helper_method: :suppressed_to_status
         config.add_facet_field solr_name("resource_type", :facetable), limit: 5
         config.add_facet_field solr_name("hyrax_batch_ingest_batch_id", :stored_searchable)
+        config.add_facet_field solr_name("bulkrax_importer_id", :stored_searchable)
```

Comment:
yes, it does.

Reply:
I seem to recall having had problems when storing IDs in `_tesim` fields because Solr would tokenize. But I don't recall the specifics, i.e. if it was due to the ID format or what. In any event, I set a flag in my brain to try to store IDs as symbols (e.g. ssim) rather than text.

Like I said, not sure if it matters here. If not, then 👍